apps/api/src/utils/resume_handler.py

```diff
@@ -11,20 +11,29 @@
 
 log = getLogger(__name__)
 
-RESUMES_FOLDER_ID = os.getenv("RESUMES_FOLDER_ID")
+HACKER_RESUMES_FOLDER_ID = os.getenv("HACKER_RESUMES_FOLDER_ID")
+MENTOR_RESUMES_FOLDER_ID = os.getenv("MENTOR_RESUMES_FOLDER_ID")
 SIZE_LIMIT = 500_000
 ACCEPTED_TYPES = ("application/pdf",)
 
 
 class Person(Protocol):
     first_name: str
     last_name: str
+    application_type: Union[Literal["Hacker"], Literal["Mentor"]]
 
 
 async def upload_resume(person: Person, resume_upload: UploadFile) -> HttpUrl:
     """Upload resume file to Google Drive and provide url to uploaded file.
     Reject files larger than size limit"""
-    if not RESUMES_FOLDER_ID:
+
+    RESUME_FOLDER_ID = None
+    if person.application_type == "Hacker":
+        RESUME_FOLDER_ID = HACKER_RESUMES_FOLDER_ID
+    elif person.application_type == "Mentor":
+        RESUME_FOLDER_ID = MENTOR_RESUMES_FOLDER_ID
+
+    if not RESUME_FOLDER_ID:
         raise RuntimeError("RESUMES_FOLDER_ID is not defined")
```

Comment:
Note for later: we probably want to move a check like this to `apps/api/index.py` and add an integration test so we ensure these environment variables are always set on a deployment.